### ISSUE TITLE
Scripting Define Symbol "NO_GPGS" does not work with Unity Build Option "Symlink Unity libraries"

### ISSUE THREAD

**Issue by slaosk** [playgameservices/play-games-plugin-for-unity#619]
The PostProcessBuild routine in `GPGSPostBuild` assumes that the GPGSAppController.mm file is located under the `Libraries` folder hierarchy when removing the code within to exclude GPGS from iOS builds.

However, this is not the case when using the Unity Build Option "Symlink Unity libraries", in which case no copy of the GPGSAppController.mm file is made and the original file located in Assets/Plugins/iOS/GPGSAppController.mm is referenced by the Xcode project. So the post process build will fail to find the file to modify, leading to an error when building in Xcode:

```
...Assets/Plugins/iOS/GPGSAppController.mm:20:9: fatal error: 'GooglePlus/GooglePlus.h' file not found
#import <GooglePlus/GooglePlus.h>
        ^
1 error generated.
```

**Comment by claywilkinson** [comment]
Thanks for the issue.  I'll take a look at how to improve it.  Do you have a fix in mind?

**Comment by slaosk** [comment]
Not really. I was thinking of setting some macros in Xcode to turn on/off the GPGSAppController but didn't get a chance to look into how to write a PostProcessBuild script in Unity to do that.

**Comment by claywilkinson** [comment]
We fixed this in 0.9.25 by reading the pbx file for the project and cleaning up all the files (regardless of where they are located).
